c++/include/orc/sargs/Literal.hh

```diff
@@ -63,10 +90,15 @@ namespace orc {
     Literal(bool val);
 
     /**
-     * Create a literal of Timestamp or DATE type
```

Comment:
This looks like a breaking change at `Literal` class. Can we create `Timestamp` literal with `Literal(PredicateDataType type, int64_t val)` like before?